lib/component.js

```diff
@@ -161,6 +161,21 @@ class Component extends WebComponent {
     }
   }
 
+  /**
+   * Returns the reference to the parent of the given name. An error is thrown out if no parent of
+   * such name is found.
+   * @param {tagName} - name of the parent to search for.  @example
+   * this.parentbyTagName('myApp');
```

Comment:
as far as examples go, this one will never work in practice; custom element names are required to have a dash in them, and browsers uppercase tag names, so this more correctly would be `this.parentbyTagName('MY-APP');` but see my previous comment about allowing `this.parentbyTagName('my-app');`